package.json

```diff
@@ -22,49 +24,51 @@
   "devDependencies": {
     "connect-livereload": "~0.6.0",
     "expect": "^1.20.2",
-    "grunt": "~0.4.1",
-    "grunt-autoprefixer": "~2.2.0",
-    "grunt-bower-install": "~0.7.0",
+    "grunt": "^1.0.1",
+    "grunt-autoprefixer": "^3.0.4",
     "grunt-cli": "^0.1.13",
-    "grunt-concurrent": "~1.0.0",
-    "grunt-contrib-clean": "~0.6.0",
-    "grunt-contrib-concat": "~0.5.1",
-    "grunt-contrib-copy": "~0.8.0",
-    "grunt-contrib-htmlmin": "~0.4.0",
-    "grunt-contrib-uglify": "~0.8.0",
-    "grunt-contrib-watch": "~0.6.1",
-    "grunt-env": "~0.4.1",
-    "grunt-express-server": "~0.4.5",
+    "grunt-concurrent": "^2.3.0",
+    "grunt-contrib-clean": "^1.0.0",
+    "grunt-contrib-concat": "^1.0.1",
+    "grunt-contrib-copy": "^1.0.0",
+    "grunt-contrib-htmlmin": "^1.4.0",
+    "grunt-contrib-sass": "^1.0.0",
+    "grunt-contrib-uglify": "^1.0.1",
+    "grunt-contrib-watch": "^1.0.0",
+    "grunt-env": "^0.4.4",
+    "grunt-express-server": "*",
     "grunt-htmlrefs": "^0.5.0",
-    "grunt-jscs": "^1.8.0",
-    "grunt-karma": "~0.10.1",
-    "grunt-newer": "~1.1.0",
-    "grunt-ng-annotate": "^0.10.0",
-    "grunt-node-inspector": "~0.1.3",
+    "grunt-jscs": "^3.0.1",
+    "grunt-karma": "~2.0.0",
+    "grunt-newer": "^1.2.0",
+    "grunt-ng-annotate": "^2.0.2",
+    "grunt-node-inspector": "~0.4.2",
     "grunt-nodemon": "~0.4.0",
-    "grunt-open": "~0.2.0",
-    "grunt-rev": "~0.1.0",
-    "grunt-sass": "^0.18.1",
-    "grunt-usemin": "~3.0.0",
+    "grunt-open": "^0.2.3",
+    "grunt-rev": "^0.1.0",
+    "grunt-usemin": "^3.1.1",
+    "grunt-sass": "^2.0.0",
+    "jasmine-core": "*",
     "jshint": "^2.9.2",
-    "karma": "~0.12.31",
+    "karma": "^1.0.0",
     "karma-chrome-launcher": "~0.1.2",
     "karma-firefox-launcher": "~0.1.3",
-    "karma-html2js-preprocessor": "~0.1.0",
-    "karma-jasmine": "~0.3.5",
-    "karma-ng-html2js-preprocessor": "~0.1.0",
-    "karma-ng-scenario": "~0.1.0",
-    "karma-phantomjs-launcher": "~1.0.2",
-    "karma-requirejs": "~0.2.1",
-    "karma-script-launcher": "~0.1.0",
-    "load-grunt-tasks": "~3.1.0",
-    "mocha": "^3.2.0",
-    "open": "~0.0.4",
-    "time-grunt": "~1.4",
+    "karma-jasmine": "^1.0.2",
+    "karma-ng-html2js-preprocessor": "^1.0.0",
+    "karma-ng-scenario": "^1.0.0",
+    "karma-phantomjs-launcher": "^1.0.1",
+    "karma-requirejs": "^1.0.0",
+    "karma-script-launcher": "^1.0.0",
+    "load-grunt-config": "^0.19.2",
+    "mocha": "^3.1.0",
+    "open": "0.0.5",
+    "phantomjs": "^2.1.7",
+    "requirejs": "^2.2.0",
+    "time-grunt": "^1.3.0",
     "watai": "^0.7.0"
   },
   "engines": {
-    "node": "0.10"
+    "node": ">=6.9.5"
```

Comment:
Have you actually checked that you don't support Node v6.8? Be as lax as possible in the version constraints: once you've stated an engine version constraint, one has to trust you and getting the specific environment for running your app can be very costly.